parler/models.py

```diff
@@ -307,11 +307,20 @@ def set_current_language(self, language_code, initialize=False):
 
     def get_fallback_language(self):
         """
+        Backwards compatible fallback language getter
+        """
+        fallbacks = self.get_fallback_languages()
+        return fallbacks[0] if fallbacks else None
+
+
+    def get_fallback_languages(self):
+        """
         Return the fallback language code,
         which is used in case there is no translation for the currently active language.
         """
         lang_dict = get_language_settings(self._current_language)
-        return lang_dict['fallback'] if lang_dict['fallback'] != self._current_language else None
+        fallbacks = [lang for lang in lang_dict['fallbacks'] if lang != self._current_language]
+        return fallbacks or None
```

Comment:
I think this should be `or []` so everything else can just loop through the list.